python/triton/runtime/autotuner.py

```diff
@@ -261,10 +272,13 @@ def kernel(x_ptr, x_size, **META):
     :type warmup: int
     :param rep: Repetition time (in ms) to pass to benchmarking, defaults to 100.
     :type rep: int
+    :param report: Flag to enable printing the selected configuration
```

Comment:
Nit: Reword to something like what you wrote in the commit message, which is more helpful.  For example: "If `print` is true, Triton will print a log message each time it autotunes a function."

Reply:
Fair point, I updated it.